pkg/collector/python/memory_linux.go

```diff
@@ -0,0 +1,12 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+// +build python
+
+package python
+
+import (
+	_ "github.com/benesch/cgosymbolizer"
```

Comment:
I'm fine with this, though strictly speaking I think we'd want to only avoid this import for darwin (note that there are freebsd ports of the datadog-agent).

Reply:
[README says](https://github.com/benesch/cgosymbolizer#platform-support)
> Only Linux and macOS are supported.

Reply:
Yeah, I _think_ we can actually now support other platforms too, but we weren't doing it ourselves, so this is fine.